src/lib/network/network_autoyast.rb

```diff
@@ -210,32 +210,13 @@ def merge_routing(instsys_routing, ay_routing)
       instsys_routing.merge(ay_routing)
     end
 
-    # Returns current AY profile in the internal representation
-    #
-    # @return [Hash] hash representing current profile or empty hash
-    def ay_current_profile
```

Comment:
But then, there is no ay_current_profile

Reply:
yep, it is correct. Previously it was typo.